web_last_viewed_records/static/src/js/main.js

```diff
@@ -0,0 +1,178 @@
+openerp.web_last_viewed_records = function(instance){
```

Comment:
odoo's pos is an isolated piece of software that doesn't break other things. If you have a browser that doesn't support localStorage, you can't use POS, but the rest of odoo. If I install this module, everything breaks on an old browser